sentry/src/main/java/io/sentry/JsonReflectionObjectSerializer.java

```diff
@@ -131,4 +173,17 @@ public final class JsonReflectionObjectSerializer {
     }
     return hashMap;
   }
+
+  private @NotNull Map<String, Object> calendarToMap(@NotNull Calendar calendar) {
+    Map<String, Object> map = new HashMap<>();
+
+    map.put("year", (long) calendar.get(1));
```

Comment:
Maybe better to use the Calendar constants, for readability
```suggestion
    map.put("year", (long) calendar.get(Calendar.YEAR));
```
Same for the other gets

Reply:
Updated